src/Microsoft.Net.Http.Headers/NameValueHeaderValue.cs

```diff
@@ -21,6 +21,9 @@ public class NameValueHeaderValue
 
         private StringSegment _name;
         private StringSegment _value;
+        private bool _quotedValue;
```

Comment:
BTW even if we were to save it, a better name would be `_isQuotedValue`. Before I noticed this was a `bool`, I thought it stored the quoted value, not whether the value was quoted.